tests/integration/core/test_integration_neural_graph_nesting.py

```diff
@@ -0,0 +1,97 @@
+# ! /usr/bin/python
+# -*- coding: utf-8 -*-
+
+# =============================================================================
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+import pytest
+import torch
+
+from nemo.backends.pytorch.actions import PtActions
+from nemo.backends.pytorch.tutorials import MSELoss, RealFunctionDataLayer, TaylorNet
+from nemo.core import EvaluatorCallback, NeuralGraph, OperationMode, SimpleLossLoggerCallback
+from nemo.utils import logging
+
+
+@pytest.mark.usefixtures("neural_factory")
+class TestNeuralGraphNesting:
+    @pytest.mark.integration
+    def test_nesting_operation_modes_example(self):
```

Comment:
Should we be adding asserts to this test?

Reply:
the same here

Reply:
so long as neural graph nesting is tested elsewhere, I see little use for this test now apart from the fact that it tests for `RealFunctionDataLayer` and `TaylorNet`. And these two do not seem particularly useful.

Reply:
done